codalab/server/bundle_manager.py

```diff
@@ -327,7 +327,14 @@ def _deduct_worker_resources(self, workers_list):
         """
         for worker in workers_list:
             for uuid in worker['run_uuids']:
-                bundle = self._model.get_bundle(uuid)
+                try:
+                    bundle = self._model.get_bundle(uuid)
+                except NotFoundError:
+                    logger.info(
+                        'Bundle %s in WorkerInfoAccessor but no longer found. Skipping for resource deduction.',
```

Comment:
Why does this message have 'in WorkerInfoAccessor' in it?

Reply:
`workers_list` is just a list extracted from the `WorkerInfoAccessor` that we create earlier here, so the source of this is that object. This was meant to be aimed towards administrators since it will be logged server-side, so wanted to include the actual object class in the message.